src/index.html

```diff
@@ -1,13 +1,72 @@
 <!doctype html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <title>Product cards</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Product cards</h1>
-</body>
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
+          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <title>Product cards</title>
+    <link 
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+    >
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,400;0,500;0,700;1,400&display=swap"
+      rel="stylesheet"
+    > 
+    <link 
+      rel="stylesheet"
+      href="./style.css"
+    >
+  </head>
+  <body>
+    <div
+      class="card"
+      data-qa="card"
+    > 
+      <img 
+        src="images\imac.jpeg" 
+        alt="Photo of iMac product" 
+        class="card__image"
+      >
+      <div class="card__name">
+        Моноблок APPLE A1419 iMac 27" Retina 5K (MNED2UA/A)
+      </div>
+      <div class="card__code">
+        Код товара: 195434 
+      </div>
+      <div class="card__reviews">
+        <div class="card__reviews-stars">
+          <span class="card__review-star"></span>
+          <span class="card__review-star"></span>
+          <span class="card__review-star"></span>
+          <span class="card__review-star"></span>
+          <span class="card__review-star"></span>
+        </div>
+        <span class="card__reviews-text">
+          Отзывов: 5
+        </span>
+      </div>
+      <div class="card__price">
+        <span class="card__price-text">
+          Цена:
+        </span>
+        <span class="card__price-value">
+          69 999 грн
+        </span>
+      </div>
+      <div 
+      class="card__button"
+      >
+        <span>
+          <a
+          href="#"
+          class="card__button-link"
+          data-qa="hover"
+          >
```

Comment:
```suggestion
          <a
            href="#"
            class="card__button-link"
            data-qa="hover"
          >
```
Добавляй атрибутам отступ